### DISCUSSION TITLE
Connect Kinect camera to Cables

### DISCUSSION THREAD

**Discussion by Nuuk23** [cables-gl/cables_docs#515] [Answers & Questions] [answered]
Dear all, 
do any of you have experience how to connect a Kinect XBOX One to Cables? Unfortunately I'm missing the corresponding operator. There's one called "Kinect Op" but it's deprecated. The camera is already connected to my Windows laptop and all that's missing is access to Cables. Can someone help me? I would be very happy about a tip!

**Comment by KOLEKTYW-online** [comment]
I use work-around with virtual spout webcam. I connect kinect to vvvv or touchdesigner, send it via spout and use webcam texture in cables

**Comment by pandrr** [comment, marked as answer]
kinect is not accessible in the browser

you can use a different tool to expose the bone data via websockets or something like that

also try the mediapipe ops, they work quite great: https://cables.gl/op/Ops.Libs.Mediapipe.MpPoseTracking
